source/2019-04-12-the-ember-times-issue-94.md

```diff
@@ -92,65 +93,70 @@ If you'd like to learn more, please take a look at [RFC #463](https://github.com
 On a related note, [@igorT](https://github.com/igorT) proposes in [RFC #466](https://github.com/emberjs/rfcs/pull/466) that we create a **Request State service**, which returns the `in-flight` and `completed` states of a request. The service would, not only simplify how we determine the state of a request, but also help us work with data from multiple sources.
 
 You can read more about the proposal in [RFC #466](https://github.com/emberjs/rfcs/pull/466). As always, feedback is welcome! 🙂
-  
+
 ---
 
 ## [Section Title in Title Case 🐹](#section-url)
 <change section title emoji>
 <consider adding some bold to your paragraph>
-  
+
 <add your name to author list, top and bottom>
 <add blurb and emoji to "SOME-INTRO-HERE">
-  
+
 ---
 
 ## [Section Title in Title Case 🐹](#section-url)
 <change section title emoji>
 <consider adding some bold to your paragraph>
-  
+
 <add your name to author list, top and bottom>
 <add blurb and emoji to "SOME-INTRO-HERE">
-  
+
 ---
 
-## [Section Title in Title Case 🐹](#section-url)
-<change section title emoji>
-<consider adding some bold to your paragraph>
-  
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
-  
+## ["I contribute to Ember" with Danielle Adams 💬](https://discuss.emberjs.com/t/i-contribute-to-ember-with-danielle-adams/16473)
+
+<div class="float-right padded portrait-frame">
+  <img alt="Danielle Adams" title="Danielle Adams - Contributor to Ember" src="/images/blog/emberjstimes/danielleadams.jpg" />
+</div>
+
+In our 10th edition of the contributor interview series, community member **Danielle Adams**, also known as [@danielleadams](https://github.com/danielleadams), talks about her work as a **mentor**, conference **MC** and **public speaker** and shares tips to be present and confident when presenting your ideas. ✨
+
+You can read the full interview on [the Ember Forum](https://discuss.emberjs.com/t/i-contribute-to-ember-with-danielle-adams/16473).
+
+<a class="ember-button ember-button--centered" href="https://discuss.emberjs.com/t/i-contribute-to-ember-with-danielle-adams/16473">Read more</a>
```

Comment:
It looks like this link is a duplicate of the one right above?

Reply:
Hmm one says Read more and the other is part of the sentence. What do you think @jessica-jordan ?

Reply:
Yes exactly, I always add both a text link and a CTA button to the interview section trying to  get people to click either one of them and actually read the interview:

<img width="1071" alt="Screenshot 2019-04-19 at 16 42 57" src="https://user-images.githubusercontent.com/8811742/56431543-5924a380-62ca-11e9-9dca-f1b956f70231.png">

Reply:
Ah, I see. Cool.